ext/skeleton/composer.json.in

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "your-vendor/%EXTNAME%",
+    "type": "php-ext",
+    "license": "BSD-3-Clause",
```

Comment:
```suggestion
    "license": "MIT",
```

as less-stricter and the most commonly used - https://innovationgraph.github.com/global-metrics/licenses

Reply:
This was selected in anticipation of this RFC: https://wiki.php.net/rfc/php_license_update

If it does not pass, then the licence should be the PHP 3.01 licence, not MIT.

I'm not here to debate the merits of each license, but if selecting BSD-3-Clause is contentious as the RFC is not even in voting just yet, I'll change this to PHP 3.01, since that is the current recommendation: https://pecl.php.net/account-request.php

Reply:
IMO the RFC for PHP sources licensing is not binding for the skeleton ext.

I would still prefer `MIT` for the skeleton as it is very permissive and known around the world, but if PHP 3.01 is wanted by default, then it has to written with `-`:

```suggestion
    "license": "PHP-3.01",
```

src: https://spdx.org/licenses/PHP-3.01.html

Reply:
> IMO the RFC for PHP sources licensing is not binding for the skeleton ext.

It affects the recommendation for the default. Remember this is a skeleton, and extension authors are welcome to select whichever licence they prefer. This is just the default recommendation.

> I would still prefer `MIT` for the skeleton as it is very permissive and known around the world, but if PHP 3.01 is wanted by default, then it has to written with `-`:

Yes, I am aware thanks!